main.nf

```diff
@@ -168,6 +168,11 @@ workflow NFCORE_SAREK {
     versions = versions.mix(PREPARE_GENOME.out.versions)
     versions = versions.mix(PREPARE_INTERVALS.out.versions)
 
+    // Set Varlociraptor reference files
+    varlociraptor_scenario_germline   = params.varlociraptor_scenario_germline   ? Channel.fromPath(params.varlociraptor_scenario_germline).map { it -> [[id: it.baseName - '.yte'], it] }.collect()    : Channel.fromPath("${projectDir}/assets/varlociraptor_germline.yte.yaml").collect()
+    varlociraptor_scenario_somatic    = params.varlociraptor_scenario_somatic    ? Channel.fromPath(params.varlociraptor_scenario_somatic).map { it -> [[id: it.baseName - '.yte'], it] }.collect()     : Channel.fromPath("${projectDir}/assets/varlociraptor_somatic.yte.yaml").collect()
+    varlociraptor_scenario_tumor_only = params.varlociraptor_scenario_tumor_only ? Channel.fromPath(params.varlociraptor_scenario_tumor_only).map { it -> [[id: it.baseName - '.yte'], it] }.collect()  : Channel.fromPath("${projectDir}/assets/varlociraptor_tumor_only.yte.yaml").collect()
```

Comment:
Can we put that directly on the call to Sarek workflow?

Reply:
as you wish, thought it would be more new syntaxy to do it in the main?

Reply:
basically this follows where currently the intervals and annotation files are defined.